doc/query.md

```diff
@@ -72,9 +105,47 @@ if (name == null) {
 }
 ```
 
-`ensureIndexed(List<Object> fieldNames, String indexName)` returns the name of the index if it is successful, otherwise it returns `null`.
+Since text search relies on SQLite FTS, we must ensure that SQLite was compiled with both the `ENABLE_FTS3` and `ENABLE_FTS3_PARENTHESIS` options.  To verify that text search is enabled and that a text index can be created use `isTextSearchEnabled()` before attempting to create a text index.  If text search is not enabled see [compiling and enabling SQLite FTS][enableFTS] and [SQLite Android Bindings][androidBind] for details. 
```

Comment:
I would have a further sub-sub-sub-heading here because we're moving onto `text` specific stuff here.